providers/provider-springmvc/src/test/java/org/apache/servicecomb/provider/springmvc/reference/async/CseAsyncRequestCallbackTest.java

```diff
@@ -28,27 +28,24 @@
   @Test
   public void testNormal() {
     CseAsyncClientHttpRequest request = new CseAsyncClientHttpRequest();
-    @SuppressWarnings("unchecked")
-    CseAsyncRequestCallback cb = new CseAsyncRequestCallback(null);
+    CseAsyncRequestCallback<HttpEntity<String>> cb = new CseAsyncRequestCallback<HttpEntity<String>>(null);
```

Comment:
CseAsyncRequestCallback<HttpEntity<String>> cb = new CseAsyncRequestCallback<>(null);

Reply:
As HttpEntity is still a raw type here, this could cause a warning again.
how about `CseAsyncRequestCallback<HttpEntity<?>> cb = new CseAsyncRequestCallback<>(null);`

Reply:
sorry, copy lost information, i want to express "new CseAsyncRequestCallback<>(...)", no need to declare type again.